package.json

```diff
@@ -23,7 +23,7 @@
     "lint:all": "backstage-cli repo lint",
     "lint:type-deps": "backstage-repo-tools type-deps",
     "docker-build": "yarn tsc && yarn workspace example-backend build --build-dependencies && yarn workspace example-backend build-image",
-    "new": "backstage-cli new --scope backstage --baseVersion 0.0.0 --no-private",
+    "new": "backstage-cli new --scope backstage --baseVersion 0.0.0 --no-prßivate",
```

Comment:
`prßivate` looks weird. is it supposed to be like this?

Reply:
No, it isn't. Thanks